pkg/quickwit/client/client.go

```diff
@@ -166,6 +166,13 @@ func (c *baseClientImpl) ExecuteMultisearch(r *MultiSearchRequest) (*MultiSearch
 
 	c.logger.Debug("Received multisearch response", "code", res.StatusCode, "status", res.Status, "content-length", res.ContentLength)
 
+	if res.StatusCode >= 400 {
```

Comment:
We should return an error immediately.
We can improve that in a dedicated PR.

Reply:
Issue created: https://github.com/quickwit-oss/quickwit-datasource/issues/29